src/api/services/timecardService.js

```diff
@@ -19,11 +19,23 @@ export const getTimeEntries = async (params) => {
   }
 };
 
-export const saveTimeEntry = async (payload, params) => {
+export const createTimeEntry = async (payload, params) => {
   try {
     return await api.post(baseUrl + 'resources/time-entry', params, payload);
   } catch (error) {
-    throw new Error(serviceName + ' saveTimeEntry function threw ' + error);
+    throw new Error(serviceName + ' createTimeEntry function threw ' + error);
+  }
+};
+
+export const updateTimeEntry = async (id, payload, params) => {
```

Comment:
Small code formatting comment:

Ln 24 has an API call as well but is formatted different. Not sure if we have a coding style for this. But would be good if they aligned.

Reply:
Ah so this is the prettier (linting) tool that I believe breaks the line down if it exceeds 80 characters. All the other api call lines are under so it keeps them as one line.